### PR TITLE
Use a skinned model that shows off skinning

### LINKED ISSUE #24005: Should skinning examples show skinning?
I noticed none of skinning examples actually show skinning. At best maybe the stomach of the stormtrooper in [this example](https://threejs.org/examples/webgl_loader_collada_skinning.html) shows skinning. The rest might be using the skinning code but the models don't "skin". They're a collection of solid parts that would look exactly same without skinning code.

A good example that makes this clear is this example:

https://threejs.org/examples/webgpu_skinning_points.html

You can see, no points are being weight blending by multiple vertices. That's not "skinning"

The same is true for all of these examples: https://threejs.org/examples/webgpu_skinning_instancing.html, https://threejs.org/examples/webgpu_skinning.html, https://threejs.org/examples/webgl_animation_skinning_morph.html,
https://threejs.org/examples/webgl_animation_skinning_additive_blending.html

Even this one, https://threejs.org/examples/webgl_animation_skinning_blending.html, it's hard to see any actual "skinning". It's possible there's some skinning in the black parts of the joints but given the armor is stiff this example shows no actual skinning.

If it's not clear what I mean. This is skinning (computing vertex positions by summing the weighted influence of multiple matrices)

skinning-lines
skinning-solid

This is not skinning (moving independent meshes)

not-skinning-lines
not-skinning-solid

All of the three.js "skinning" examples are not skinning or at least not good examples of skinning. Note: It's possible the examples are going through the skinning code. It's even possible they're using weights but if so the weights are all set such that there's only one matrix influencing each vertex which is not really a good demonstration of skinning.

Would it be ok to replace the example models with something else? A quick search, all of these CC-BY models are much better examples of skinning. 

https://sketchfab.com/3d-models/tyrannosaurus-rex-20-512712b314404760a860389ebd0ce78a

https://sketchfab.com/3d-models/white-pointer-8e429052939a4677861d0d550a0e27cd

https://sketchfab.com/3d-models/hummingbird-flying-4e6d591f6e50493aa5e31355084fc4e8

https://sketchfab.com/3d-models/ramsey-fb31d1eeeb90425394d0a708bd5e4552

https://sketchfab.com/3d-models/elephant-animation-idle-a8e7e10f005f4baab0a2f5079d759fcd

https://sketchfab.com/3d-models/icy-dragon-2db9268227b943e6a41e88390f2875a6

https://sketchfab.com/3d-models/low-poly-fox-by-pixelmannen-animated-371dea88d7e04a76af5763f2a36866bc

https://sketchfab.com/3d-models/fox-rigged-ee7d17f164724896b4f2c417a5153163

https://sketchfab.com/3d-models/tiger-51ed5186afb04487ae6adb51f8ffd09b

https://sketchfab.com/3d-models/animated-dog-shiba-inu-9abfce885a834399b2c3ccaed51cd474

I'd be happy to replace the current models with some of these.

### PR DESCRIPTION
**Pull request by greggman**
Fixed #24005.

**Description**

Update a few skinning examples with a model that shows off skinning more clearly

### CODE REVIEW

File: examples/webgpu_skinning.html [hunk 1]
```diff
@@ -54,38 +54,27 @@
...
+				loader.load( 'models/gltf/Michelle.glb', function ( gltf ) {
 
+					const object = gltf.scene;
 					mixer = new THREE.AnimationMixer( object );
 
-					const action = mixer.clipAction( object.animations[ 0 ] );
+					const action = mixer.clipAction( gltf.animations[ 0 ] );
 					action.play();
 
-					object.traverse( function ( child ) {
```
Comment (LeviPesin) [inline examples/webgpu_skinning.html:78]: Why did you remove this?

Comment (greggman) [reply examples/webgpu_skinning.html:78]: so the character would show its materials.

With those lines

Screenshot 2023-03-11 at 08 30 53

Without those lines

Screenshot 2023-03-11 at 08 29 32

Comment (LeviPesin) [reply examples/webgpu_skinning.html:78]: OK. Please remove `MeshStandardNodeMaterial` import then.

Comment (greggman) [reply examples/webgpu_skinning.html:78]: done

**Comment by donmccurdy** [comment]
Shaved a bit off the file size, 3.1 MB → 1.3 MB:

[Michelle.glb.zip](https://github.com/mrdoob/three.js/files/10949176/Michelle.glb.zip)

**Comment by greggman** [comment]
@donmccurdy, what did you do? what tools/options do you use?

**Comment by donmccurdy** [comment]
```
npm install --global @gltf-transform/cli

gltf-transform optimize input.glb output.glb \
  --simplify 0 \
  --compress quantize \
  --texture-compress webp
```

**Comment by greggman** [comment]
sadly it apparently uncovered a bug in the Three WebGPU support. With the compressed file I get

```
Attribute offset (24) with format VertexFormat::Float32x4 (size: 16) doesn't fit in the vertex buffer stride (28).
 - While validating attributes[0].
 - While validating buffers[2].
 - While validating vertex state.
 - While calling [Device].CreateRenderPipeline([RenderPipelineDescriptor]).

webgpu_skinning.html:1
```

**Comment by donmccurdy** [comment]
Ah, possibly we don't support interleaved vertex attributes in WebGPU yet? This version should work if that's the cause, I added `--vertex-layout separate`. 

[Michelle.glb.zip](https://github.com/mrdoob/three.js/files/10949496/Michelle.glb.zip)

If that's not fixing it no worries, feel free to keep the existing model!

**Comment by greggman** [comment]
That one got a similar error.

> Attribute offset (0) with format VertexFormat::Float32x3 (size: 12) doesn't fit in the vertex buffer stride (8).
> - While validating attributes[0].
> - While validating buffers[0].
> - While validating vertex state.
> - While calling [Device].CreateRenderPipeline([RenderPipelineDescriptor]).

I'll use the bigger file for now. I don't know if I have time to fix the WebGPU code. I'm sure it's more complicated but the error is straight forward so maybe there's a possibility that it's a simple fix.

**Comment by LeviPesin** [comment]
Seems like the issue is in [this](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/renderers/webgpu/WebGPUBufferUtils.js) file.

**Comment by LeviPesin** [comment]
Related: https://github.com/mrdoob/three.js/pull/25249
I'm not sure how much correct was that fix and could a similar one be applied here... You should probably create a new issue.

**Comment by greggman** [comment]
AFAICT there's a normalized Int16x3 attribute in the file. WebGPU only supports 32bit formats x3. It doesn't support any 16 bit formats x3 nor x1, nor 8bit formats by x3 nor x1

Two workarounds would be to either, at load time, convert the data to something WebGPU supports, or stop using attributes and use storage buffers and pull the data out in the shaders.

**Comment by donmccurdy** [comment]
Ok, sorry for the trouble! Ignore the compressed model for now. I'll have to look into what we can do with WebGPU, maybe need the tool to have an option padding the buffer for the correct byte stride.

**Comment by greggman** [comment]
> maybe need the tool to have an option padding the buffer for the correct byte stride.

I'd think you'd want GLTFLoader to load any GLTF and not just padded ones. I'm not sure where is best to fix it. In the GLTFLoader (in which case it would have to be aware that it's loading for WebGPU vs WebGL) or in the WebGPURender (trying to make it either support 8bit and 16bit x3 and x1) directly via shader contortions or trying to make it do the conversion.

My guess would be it'd be best to make the loader do this. That would keep the renderer simpler

**Comment by donmccurdy** [comment]
Ah, no I wouldn't want to change the renderer here either. And ideally GLTFLoader can just copy buffers without repacking them. I'm thinking perhaps glTF Transform needs to have an additional vertex layout option, when quantizing geometry for WebGPU.
